router/batchrouter/asyncdestinationmanager/marketo-bulk-upload/authService.go

```diff
@@ -0,0 +1,67 @@
+package marketobulkupload
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+)
+
+type MarketoAccessToken struct {
+	AccessToken string `json:"access_token"`
+	TokenType   string `json:"token_type"`
+	ExpiresIn   int64  `json:"expires_in"`
+	FetchedAt   int64
```

Comment:
is it by choice that there is no json tag here?

Reply:
Yes it is intentional